ssh/tunnel.go

```diff
@@ -177,6 +179,19 @@ func (t *Tunnel) Stop() error {
 	return nil
 }
 
+func (t *Tunnel) startKeepalive(user string, sshConnection *ssh.Client) {
+	ticker := time.NewTicker(2 * time.Second)
```

Comment:
I wonder whether this needs to be as frequent as this. Something like 30secs is probably plenty...

Reply:
The closest I could find to any documentation on this feature is in `man ssh_config` which has these options:

```
     ServerAliveCountMax
             Sets the number of server alive messages (see below) which may be sent without ssh(1) receiving any messages back from the server.  If this threshold
             is reached while server alive messages are being sent, ssh will disconnect from the server, terminating the session.  It is important to note that the
             use of server alive messages is very different from TCPKeepAlive (below).  The server alive messages are sent through the encrypted channel and there-
             fore will not be spoofable.  The TCP keepalive option enabled by TCPKeepAlive is spoofable.  The server alive mechanism is valuable when the client or
             server depend on knowing when a connection has become inactive.

             The default value is 3.  If, for example, ServerAliveInterval (see below) is set to 15 and ServerAliveCountMax is left at the default, if the server
             becomes unresponsive, ssh will disconnect after approximately 45 seconds.

     ServerAliveInterval
             Sets a timeout interval in seconds after which if no data has been received from the server, ssh(1) will send a message through the encrypted channel
             to request a response from the server.  The default is 0, indicating that these messages will not be sent to the server.
```

>   The default value is 3.  If, for example, ServerAliveInterval (see below) is set to 15...

So maybe 15 seconds is a better hardcoded value 🤔

Reply:
Updated to use 15 secs.